src/lib/swapper/swappers/ThorchainSwapper/constants.ts

```diff
@@ -49,3 +49,7 @@ export const ALLOWANCE_CONTRACT = '0xF892Fef9dA200d9E84c9b0647ecFF0F34633aBe8' /
 
 export const UTXO_MAXIMUM_BYTES_LENGTH = 80
 export const BCH_MAXIMUM_BYTES_LENGTH = 220
+
+export const AGGREGATOR_EXPONENT_LENGTH = 2
+export const AGGREGATOR_MAXIMUM_EXPONENT = 99
+export const THORCHAIN_PARSER_MAXIMUM_PRECISION = 18
```

Comment:
Do we want to add comments for these to make it clearer what these do for someone without L1-to-longtail magic context?

Reply:
Also while I'm all for dedicated constants modules for reusibility, looks like this is pretty much isolated to the one util (`THORCHAIN_PARSER_MAXIMUM_PRECISION`) so keeping these there and colocating, along with comments, might make things easier to grasp

Reply:
I did move them back to `makeMemoWithShortenedFinalAssetAmount` with some comments